packages/core/src/lib/database/migrations/1746005191501-CreateEmployeeRecentVisitTable.ts

```
@@ -0,0 +1,287 @@
+import { DatabaseTypeEnum } from '@gauzy/config';
+
+export class CreateEmployeeRecentVisitTable1746005191501 implements MigrationInterface {
+	name = 'CreateEmployeeRecentVisitTable1746005191501';
+
+	/**
+	 * Up Migration
+	 *
+	 * @param queryRunner
+	 */
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)